models/feature-flag-user-mapping.go

```diff
@@ -9,3 +9,8 @@ type FeatureFlagUserMapping struct {
 	UpdatedAt int64  `json:"updatedAt" dynamodbav:"updatedAt"`
 	UpdatedBy string `json:"updatedBy" dynamodbav:"updatedBy"`
 }
+
+type CreateUserMapping struct {
+	Status string `json:"status" validate:"required"`
+	UserId string `json:"userId" validate:"required"`
```

Comment:
UserId we also pass in the request params, was thinking do we need this as mandatory @shubham-y ?

Reply:
Here we store the userId in the createdBy column. So I think we need to maintain the createdBy data